pkg/scheduler/event_handler.go

```diff
@@ -264,3 +283,7 @@ func schedulerNameFilter(schedulerNameFromOptions, schedulerName string) bool {
 
 	return schedulerNameFromOptions == schedulerName
 }
+
+func specificationChanged(oldObj, newObj *unstructured.Unstructured) bool {
+	return oldObj.GetGeneration() != newObj.GetGeneration()
+}
```

Comment:
I tend to in-line this `one-line` function. And the following test is essentially testing `.GetGeneration()`, we might don't need it.